scripts/billing/README.md

```diff
@@ -0,0 +1,31 @@
+# Billing Analytics - BigQuery SQL Examples
+
+In addition to the Billing product available within the GCP Console, you may want to export billing data to BigQuery for custom analytics.  Below are some example queries to help get you started using the standard data export format.  See the [links below](#useful-links) to learn more about GCP billing datasets including how to export your own billing data for analysis.
+
+The example SQL snippets below reference a sample billing export available as a public dataset in BigQuery (**`data-analytics-pocs.public.billing_dashboard_export`**).  You will need to replace references to this table with your own dataset. 
```

Comment:
Change to `bqutil.billing` once sample data is migrated.

Reply:
Migrated dataset and updated queries/readme